test/requirements/python-requirements.spec.js

```diff
@@ -0,0 +1,28 @@
+/*jshint node:true, mocha:true*/
+
+'use strict';
+const { spawnSync } = require('child_process');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+describe('python-test', function () {
+    it('Should find python 3', (done) => {
+        const processOutput = spawnSync(`python3`, ['-c', `"print('hello world')";`]);
+        assert(processOutput.status === 0);
```

Comment:
This is fine but `assert.equal(processOutput.status, 0)` might give a better default error message.

Reply:
(I still may have some asserts like this in the code base but have been trying to move over to this style myself :) )

Reply:
Ok. That should be a standard practice of using assert.